homeassistant/components/switchbot/switch.py

```diff
@@ -27,20 +28,21 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     """Perform the setup for Switchbot devices."""
     name = config.get(CONF_NAME)
     mac_addr = config[CONF_MAC]
-    add_entities([SwitchBot(mac_addr, name)])
+    password = config[CONF_PASSWORD]
```

Comment:
Please use config.get(CONF_PASSWORD) for optional config keys which do not have a default value. Moreover, dict.get() returns None if no value is set.

Reply:
Alright thanks! The changes are done.